### PR TITLE
Upgrade wayland_protocols to 1.13

### PR DESCRIPTION
**Pull request by lovedheart**
Required for weston build

#889

### CODE REVIEW

File: packages/wayland_protocols.rb [hunk 1]
```diff
@@ -3,28 +3,21 @@
...
-       i686: '11d26d2ca2b5e960b71e44573ebeba6c601e3e9e3d0e4614972c66cd8316145e',
-     x86_64: '5c1f8913e2ff2ee59dc8080b8c076383caf693890b20bbf08113d08f7fb6199e',
+
   })
 
   depends_on 'libwayland'
   
   def self.build
     system "./configure"
-    system "make"
```
Comment (uberhacker) [inline packages/wayland_protocols.rb]: Why was this line removed?

File: packages/wayland_protocols.rb [hunk 2]
```diff
@@ -3,28 +3,21 @@
...
-     armv7l: 'b1c7bf17d30ff2335dafdbd2c57bb8769bfe54f8a2342d70af06fbf0a2adc51f',
-       i686: '11d26d2ca2b5e960b71e44573ebeba6c601e3e9e3d0e4614972c66cd8316145e',
-     x86_64: '5c1f8913e2ff2ee59dc8080b8c076383caf693890b20bbf08113d08f7fb6199e',
+
   })
 
   depends_on 'libwayland'
   
   def self.build
     system "./configure"
```
Comment (uberhacker) [inline packages/wayland_protocols.rb]: Don't we need `--prefix=#{CREW_PREFIX}` and `--libdir=#{CREW_LIB_PREFIX}` configure options?

Comment (lovedheart) [reply packages/wayland_protocols.rb]: It works without it. But I can add it.

**Comment by uberhacker** [comment]
Thank you for the new package @lovedheart!  Please see inline comments.

**Comment by uberhacker** [comment]
Most packages will install fine simply with `make install` but it is a "best practice" if you run `make` prior to install.
